src/scss/_button-group.scss

```diff
@@ -0,0 +1,44 @@
+@import "scss/button-group";
+
+.btn-group {
```

Comment:
@fuhton Added in the class for that along with tests and html. Are we supporting dropdown buttons for vertical button groups? Reference links on the Issue.